internal/workflow_shadower.go

```diff
@@ -0,0 +1,308 @@
+// Copyright (c) 2017-2020 Uber Technologies Inc.
```

Comment:
maybe update the LICENSE file to 2017-2021

Reply:
Good catch. I also added license header to the query_builder files I committed in the previous diff. 

I will send out a separate diff to update the LICENSE file.

Reply:
https://github.com/uber-go/cadence-client/pull/1059